sdk/purview/Azure.Analytics.Purview.Share/README.md

```diff
@@ -59,15 +59,235 @@ We guarantee that all client instance methods are thread-safe and independent of
 
 The following section shows you how to initialize and authenticate your client and get a sent share.
```

Comment:
The following section shows you how to initialize and authenticate your client and share data.